ceph-releases/ALL/rhel8/daemon-base/__ISCSI_PACKAGES__

```diff
@@ -0,0 +1 @@
+tcmu-runner ceph-iscsi-config ceph-iscsi-cli
```

Comment:
Would you please change `ceph-iscsi-config ceph-iscsi-cli` to `ceph-iscsi` ?

Reply:
Done.